lib/builder/ProjectBuildContext.js

```diff
@@ -0,0 +1,52 @@
+const ResourceTagCollection = require("@ui5/fs").ResourceTagCollection;
+
+const STANDARD_TAGS = Object.freeze({
+	OmitFromBuildResult: "ui5:OmitFromBuildResult"
```

Comment:
The only thing left to agree on: Naming of this tag

@matz3 and I went on to rename it from `HideFromBuildResult` to `OmitFromBuildResult` after the last planning meeting. Maybe there are still other opinions.

CC: @ecker